packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,204 @@
+
+`PrimaryAction` and `SecondaryAction` will accept objects with the following keys:
+
+### PrimaryAction
+
+Renders and accepts a subset of `Button` props as keys:
+
+| Key       | Type       | Default     | Description | Required |
+| --------- | ---------- | ----------- | ----------- | -------- |
+| `text`    | `string`   | `undefined` | Content     | ✅       |
+| `onClick` | `function` | `undefined` | Callback    |          |
+
+### SecondaryAction
+
+Renders and accepts a subset of `Link` (button) props as keys:
+
+| Key       | Type       | Default     | Description | Required |
+| --------- | ---------- | ----------- | ----------- | -------- |
+| `text`    | `string`   | `undefined` | Content     | ✅       |
+| `href`    | `string`   | `undefined` | Hyperlink   |          |
+| `onClick` | `function` | `undefined` | Callback    |          |
+
+There'll also be an internal `BaseAlert` component that handles rendering web or native version similar to other Blade components. API would be similar to `Alert`.
+
+## a11y
+
+Web:
+
+- `notice` and `negative` variants will have an [`alert` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role), this is to communicate important and time sensitive message. Additionally for `notice` variant we'll set [`aria-live="polite"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) to not interrupt the user
+- Other variants will have a [`status` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role) which is for non critical changes
+- The dismiss button will have [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) as "Dismiss alert"
+
+Native:
+
+- Will set [`accessibilityRole`](https://reactnative.dev/docs/accessibility#accessibilityrole) as `alert`
+- The dismiss button will have `accessibilityLabel` as "Dismiss alert"
+
+## Usage
+
+Some example usage patterns of `Alert`.
+
+### Without title
+
+<img src="./alert-without-title.png" alt="" width="520">
+
+```jsx
+<Alert
+  variant="notice"
+  description="The payment was made 6 months ago, therefore you can’t issue refund to this merchant."
+/>
+```
+
+### Full bleed
+
+> **Note**
+>
+> Design update pending to remove all borders and border radii
+
+This full bleed layout works for all desktop, mobile and native when the `isBorderless` prop is passed
+
+<img src="./alert-mobile-full-bleed.png" alt="" width="520">
+
+```jsx
+<Alert
+  title="International Payments Only"
+  isBorderless
+  description={
+    <>
+      Use vendor payouts to quickly generate invoices.{' '}
+      <Link href="https://razorpay.com">Know More</Link>
+    </>
+  }
+/>
+```
+
+## Open questions
+
+**Q1.** Should we use a different variant naming convention in code like `success`, `warn`, `info`, `error`? Currently this is same as design
```

Comment:
We should stick to design since that's the nomenclature we follow on tokens and rest of the action level components (Check intent prop for Button & Link)

Reply:
But having said that, this is not intent. This is variant - we can choose to go ahead with something else here if it makes sense

Reply:
Same feelings, added the summary in the corresponding answer **A1**